README.md

```diff
@@ -14,15 +12,15 @@ These are all the available environment variables, along with some example value
 
 | Environment Varialbe | Example Value | Description |
 | -------------------- | ------------- | ----------- |
-| WATTS |  1500 | if your ups doesn't have NOMPOWER, set this to be the rated max power, if you do have  NOMPOWER, don't set this variable |
-| APCUPSD_HOST |  192.168.1.100 | host running apcupsd |
-| INFLUXDB_HOST |  192.168.1.101 | host running influxdb |
-| HOSTNAME |  unraid | host you want to show up in influxdb, optional defaults to apcupsd-influxdb-exporter |
-| INFLUXDB_DATABASE |  apcupsd | db name for influxdb. optional, defaults to apcupsd |
+| WATTS |  1000 | if your ups doesn't have NOMPOWER, set this to be the rated max power, if you do have NOMPOWER, don't set this variable |
+| APCUPSD_HOST | 192.168.1.100 | host running apcupsd |
```

Comment:
I think this is technically optional and it defaults to the value of INFLUXDB_HOST